trieste/models/gpflow/sampler.py

```diff
@@ -412,11 +414,11 @@ def resample_trajectory(self, trajectory: TrajectoryFunction) -> TrajectoryFunct
         return trajectory  # return trajectory with resampled weights
 
     @abstractmethod
-    def _prepare_weight_sampler(self) -> Callable[[int], TensorType]:  # [B] -> [L, B]
+    def _prepare_weight_sampler(self) -> Callable[[int], TensorType]:  # [B] -> [B, F, L]
```

Comment:
was that wrong before?

Reply:
or did we use L for the number of features?

Reply:
The comment was wrong before. The dimensions were swapped and it was using `L` for number of features. It should have been `[B, F]`, or similar.